app/views/templates/Layout.scala.html

```diff
@@ -88,7 +88,6 @@
         phaseBanner = Some(standardBetaBanner(url = appConfig.betaFeedbackUnauthenticatedUrl)),
         userResearchBanner = Some(UserResearchBanner(url="/manage-pension-schemes/user-research-banner-psa"))
     )),
-    cspNonce = CSPNonce.get,
```

Comment:
We're still importing CSPNonce in this file, should we be removing this?

Reply:
removed unused imports